docs/guide-ssl-setup.md

```diff
@@ -36,15 +36,18 @@ order: 10
 
 Report [ZABAPGIT_TEST_SSL](./other-test-ssl.html) can be used to verify that the connection works.
 
-Also see blogs
-* http://scn.sap.com/people/jens.gleichmann/blog/2008/10/31/calling-webservices-from-abap-via-httpsssl-with-pfx-certificates
-* http://scn.sap.com/people/thomas.jung/blog/2004/11/17/bsp-a-developers-journal-part-xiv--consuming-webservices-with-abap
+Also see the following blog posts on SCN:
+* [Calling WebServices from ABAP via HTTPS/SSL with pfx certificates](http://scn.sap.com/people/jens.gleichmann/blog/2008/10/31/calling-webservices-from-abap-via-httpsssl-with-pfx-certificates)
+* [BSP a Developer’s Journal Part XIV – Consuming WebServices with ABAP](http://scn.sap.com/people/thomas.jung/blog/2004/11/17/bsp-a-developers-journal-part-xiv--consuming-webservices-with-abap)
 
 The certificate list should end up looking something like this:
 ![](img/strust.png)
 
+### Note on private GitHub repositories
+Private Repositories are supported but the login process requires the certificates for [https://**api**.github.com](https://api.github.com), too. Repeat the download and STRUST import steps as described in the sections above accordingly. (See also [Issue#1491](https://github.com/larshp/abapGit/issues/1491))
```

Comment:
Just a small note:
The reason for the use of the api-certificate is not the repository being private but the operation requiring authentication. On public repositories you also need the certificate if you do any non-anonymous action like pushing commits or creating a branch.
The technical reason is that only api.github.com can supply the information whether two factor authentication is required or not. See #536

Reply:
I added a new commit that summarizes your comment.